alica_engine/include/engine/Assignment.h

```diff
@@ -87,6 +87,7 @@ class Assignment
     bool isValid() const;
     bool isSuccessful() const;
     bool isAnyTaskSuccessful() const;
+    bool isAgentSuccessful(AgentIDConstPtr id, const EntryPoint* ep) const;
```

Comment:
const AgentIDConstPtr& id

Reply:
AgentIDConstPtr is already a wrapper around a pointer to const type, so it should be passed by value.

Reply:
yup, what he said

Reply:
missed that